### ISSUE TITLE
Switching back to no language file on click

### ISSUE THREAD

**Issue by magglomag** [Antonio-Laguna/jqueryTranslator#10]
Hi,

I am using the plugin in a language switcher. My default language is German (the German strings are in my `index.html` like `<h1 data-translate="title">Title in German</h1>`) and I have a language file `content-en.js` for the English translation. 

The language can be switched on click:

```
    $buttonEN.click(function( e ) {
      $( '[data-translate]' ).jqTranslate( 'content', { forceLang: 'en' });
    });
```

What I now like to achieve is that when I click on `$buttonDE` not the German language file should be loaded but the normal strings in my `index.html`. So is there a function to just stop the plugin from loading a language file?

```
    $buttonDE.click(function( e ) {
      /* The regular strings in my `index.html` should be loaded. */
    });
```

Thanks for your help!

**Comment by Antonio-Laguna** [comment]
Hey, this is currently not possible since text is just replaced and hence, lost. It'd be a good idea to save that text as current language so you can achieve what you imply here.

I _might_ work on this as I'd like to update this plugin to use newer tools and techniques but I can't promise when that might happen.

**Comment by magglomag** [comment]
Alright, thanks for the info! 
So you mean to save the strings in the `index.html` in a file named `content-de.json`?

**Comment by Antonio-Laguna** [comment]
No, I mean saving it in memory so if the button for German is pressed, the content can be restored.

**Comment by magglomag** [comment]
Can you explain that a bit further – I don’t know what you mean with saving it in memory. Thanks!

**Comment by magglomag** [comment]
May I ask if you could give me a hint on that?
